pkg/build/builder/util/dockerfile/instructions.go

```diff
@@ -33,6 +33,11 @@ func Label(m []KeyValue) (string, error) {
 	return keyValueInstruction(command.Label, m)
 }
 
+// Run builds a RUN Dockerfile instruction from the string cmd.
+func Run(cmd string) (string, error) {
+	return unquotedArgsInstruction(command.Run, cmd)
+}
```

Comment:
is this being used somewhere?

Reply:
Yes, pkg/build/builder/docker.go:452

Reply:
oh i see. passing functions around as args. fancy.

Reply:
I wish that I could take credit for that, but it was an existing pattern in this file.